src/shared/meson.build

```diff
@@ -36,6 +36,7 @@ shared_sources = files('''
         dropin.h
         efivars.c
         efivars.h
+        enable-mempool.c
```

Comment:
Wouldn't this include the constructor with the users of the static version? AFAICT that is not the desired result.